src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const obj of actions) {
```

Comment:
use more meaningful variable names
```suggestion
  for (const action of actions) {
```